inginious-problems-blockly/templates/box_blockly.html

```diff
@@ -99,20 +95,23 @@ <h4 class="modal-title">$blockly_dico["name"]</h4>
 ></textarea>
 
 <!-- Blockly official JavaScript -->
-<script src="$get_homepath()/plugins/blockly/static/blockly/blockly_compressed.js" type="text/javascript" charset="utf-8"></script>
-<script src="$get_homepath()/plugins/blockly/static/blockly/blocks_compressed.js" type="text/javascript" charset="utf-8"></script>
-<script src="$get_homepath()/plugins/blockly/static/blockly/python_compressed.js" type="text/javascript" charset="utf-8"></script>
-<script src="$get_homepath()/plugins/blockly/static/blockly/javascript_compressed.js" type="text/javascript" charset="utf-8"></script>
+<script src="{{get_homepath(True)}}/plugins/blockly/static/blockly/blockly_compressed.js" type="text/javascript" charset="utf-8"></script>
```

Comment:
Do you actually need to set ``ignore_session`` to ``True`` ? Those are public static files, but resources requiring authentication will fail in an LTI iframe.